Tests/Marketplace/upload_packs.py

```diff
@@ -374,6 +374,7 @@ def create_corepacks_config(storage_bucket: Any, build_number: str, index_folder
     corepacks_json_path = os.path.join(artifacts_dir, GCPConfig.CORE_PACK_FILE_NAME)
     core_packs_data = {
         'corePacks': core_packs_public_urls,
+        'upgradeCorePacks': GCPConfig.CORE_PACKS_LIST_TO_UPDATE,
```

Comment:
Please add a suitable docstring for this function regarding the upgradeCorePacks part

Reply:
Done